mapsforge-core/src/main/java/org/mapsforge/core/mapelements/PointTextContainer.java

```diff
@@ -31,16 +31,14 @@ public abstract class PointTextContainer extends MapElementContainer {
     protected static final boolean DEBUG_CLASH_BOUNDS = false;
 
     public final double horizontalOffset;
-    public final boolean isVisible;
+    public final boolean isNotVisible;
```

Comment:
@Sublimis `isVisible` is more understandable and already used. Why change it?

Reply:
Two reasons: 1) `isVisible()` was only ever used in negation, cluttering the code, 2) `isVisible()` returning `true` doesn't really mean much, and can be misleading, because it does not mean the item is visible like the name implies. Like the javadoc of the superclass method `isNotVisible()` says: "@return {@code true} if this is definitely not visible; {@code false} if it may or may not be visible"